includes/class-amp-travel-cpt.php

```diff
@@ -231,6 +296,12 @@ public function save_adventure_post() {
 			if ( isset( $_POST['amp_travel_end_date'] ) ) {
 				update_post_meta( $post->ID, 'amp_travel_end_date', sanitize_text_field( wp_unslash( $_POST['amp_travel_end_date'] ) ) );
 			}
+			if ( isset( $_POST['amp_travel_images'] ) ) {
+				$images = array_map( 'floatval', wp_unslash( $_POST['amp_travel_images'] ) );
```

Comment:
Nice use of `array_map()` here. How about `intval` in place of `floatval`? I think the IDs should be integers.

Reply:
Today, I learned that`intval` is a thing! Nice!